redhat/noboolean/cassandra.spec

```diff
@@ -69,7 +69,7 @@ Cassandra is a distributed (peer-to-peer) system for the management and storage
 %build
 export LANG=en_US.UTF-8
 export JAVA_TOOL_OPTIONS="-Dfile.encoding=UTF-8"
-ant clean jar -Dversion=%{upstream_version}
+ant jar -Dversion=%{upstream_version} -Dno-checkstyle=true -Drat.skip=true -Dant.gen-doc.skip=true
```

Comment:
these scripts now intentionally don't clean. so that they can be used (sequentially) together.

for example the jenkins impl on top uses a stash from the first build, and unstashes that onto every subsequent build/test.

i haven't thought too much about the deb/rpm building, but figured constraints of "only building it from clean" should generally be implied and imposed by the caller. 🤷